src/alert/alert.scss

```diff
@@ -16,53 +17,64 @@
     color: t(iui-text-color);
     background-color: t(iui-color-background-1);
   }
-
-  &-icon {
-    @include iui-icons-default;
-    flex-shrink: 0;
-    margin-left: $iui-m;
-  }
-
-  &-message {
-    margin: $iui-baseline $iui-m;
-  }
-
-  &-link {
-    @include iui-anchor-underline('on-initial');
-    border-radius: $iui-border-radius;
-    cursor: pointer;
-    margin-left: $iui-s;
-    user-select: none;
-    white-space: nowrap;
-  }
-
-  > .iui-button {
-    margin-left: auto;
-    margin-right: $iui-s;
+  @each $status in positive, negative, warning, informational {
```

Comment:
This is to generate the `.iui-alert.iui-status` css for each of the alert statuses.